Comparison/src/cz/fidentis/comparison/localAreas/Area.java

```diff
@@ -0,0 +1,133 @@
+package cz.fidentis.comparison.localAreas;
```

Comment:
Je to v podste to, co je v development branchy v packagi cz.fidentis.comparison.hausdorffDistance.localAreas? Pokial ano, mohol by si tuto triedu aj tie ostatne prehodit do daneho packagu, alebo zrusit hausdorffDistance.localAreas package, nech to tam nemame 2x?

Reply:
odstranil som hausdorffDistance.localAreas package kedze sa nikde nepouzival